shop/static/shop/css/dialog.scss

```diff
@@ -42,7 +42,7 @@ button[shop-dialog-proceed] {
 	display: inline-block;
 }
 
-.shop-address-location {
+.shop-address-city {
```

Comment:
If we change "location" to "city" here, we should also change it a few lines above ("contain the fields zip_code and location as shown").